community/cypher/cypher-compiler-3.2/src/main/scala/org/neo4j/cypher/internal/compiler/v3_2/Comparer.scala

```diff
@@ -40,6 +40,18 @@ trait Comparer extends CypherSerializer {
         throw new IncomparableValuesException(serializeWithType(l), serializeWithType(r))
```

Comment:
Good point. I didn't want to touch orderability in this particular PR, because I know you guys are working on it, but the specified behaviour is to _never_ throw, so this code is most definitely wrong.